File: 3412-permutation-difference-between-two-strings/3412-permutation-difference-between-two-strings.py
class Solution:
    def findPermutationDifference(self, s: str, t: str) -> int:
        S = list(s)
        T = list(t)
        summ = 0
        for i in range(len(s)):
            summ+=abs(i - T.index(S[i]))
        return summ
        

File: 3412-permutation-difference-between-two-strings/3412-permutation-difference-between-two-strings.js
/**
 * @param {string} s
 * @param {string} t
 * @return {number}
 */
var findPermutationDifference = function(s, t) {
    let S = s.split('')
    let T = t.split('')
    let summ = 0

    for (let i=0; i<s.length; i++) {
        let s_char = S[i]
        summ += Math.abs(i-T.indexOf(s_char))
    }
    return summ
    
};